### ISSUE TITLE
Display name instead login in account switcher. 

### ISSUE THREAD

**Issue by kacpi2442** [shymmq/librus-client#59]

**Comment by xdk78** [comment]
IMO jest dobrze tak jak jest

**Comment by kacpi2442** [comment]
No, nie wiem czy przy pięciu kontach ogarniesz które jest które.

**Comment by shymmq** [comment]
Kto normalny ma pięć kont

**Comment by kacpi2442** [comment]
No, może przesadziłem. Ale należy zwrócić uwagę na fakt że z appki mogą też korzystać rodzice, gdzie nieraz zdarza się że nie jedno z ich dzieci posiada librusa. A często jest to typ ludzi którzy nawet nie pamiętają swojego hasła do maila, więc wątpię czy będą kojarzyli login typu 3463128.

**Comment by shymmq** [comment]
ok - ale może najpierw dodajmy opcję dodania wielu kont 😂

**Comment by kacpi2442** [comment]
Też jakiś pomysł xD
